05-JWT-Basics/preferred/errors/unauthenticated.js

```diff
@@ -0,0 +1,12 @@
+const CustomAPIError = require("./custom-error")
+const { StatusCodes } = require("http-status-codes");
+
+const UnauthenticatedError = require("./custom-error");
```

Comment:
Requires change:
You're requiring this (`"./custom-error"`) import twice. Also, you're also exporting the `UnauthenticatedError` class instead of the `BadRequest` class.